hazelcast-client/src/main/java/com/hazelcast/client/spi/impl/ClientExecutionServiceImpl.java

```diff
@@ -20,14 +20,18 @@
 import com.hazelcast.core.ICompletableFuture;
```

Comment:
Don't know exactly yet. The client also implements `Executor` and has an "internal" and "external" executor, the member has some stuff to register and retrieve named executors. So there are some extensions, but maybe we can unify those classes in the end, even if some functionality is just used in either member or clients.